Cleaned up Gergo/Georgi (#46)
qmk/qmk.fm_old: _pages/keyboards/gergo.md

{% include badge.html path="gergo" %}

![Gergo!](/keyboards/gergo/top.jpg)

> Gergo is a 14x4 ortholinear, 50-key keyboard kit, made by [g Heavy Industries](http://gboards.ca). Supporting MX, Alps and Choc switches and with a onboard processor it's one of the lowest profile QMK boards available at the moment. With fullsize mods and a thin profile, Gergo is a great addition to your desk or your bag!

Reviewer: ```suggestion
Gergo is a 14x4 ortholinear, 50-key keyboard kit, made by [g Heavy Industries](http://gboards.ca). Supporting MX, Alps and Choc switches and with an onboard processor, it's one of the lowest profile QMK boards available at the moment. With fullsize mods and a thin profile, Gergo is a great addition to your desk or your bag!
```

## Gadgets

> Gergo comes suited for gadgets to be added to the keyboard. Currently a 1u Trackball is avalible and others

Reviewer: `s/avalible/available/`

```suggestion
Gergo comes suited for gadgets to be added to the keyboard. Currently a 1u Trackball is available and others
```

are on the way! Modify your rules.mk to enable/disable specfic gadgets.

> Or if you're impaitent, build your own using the open source footprint avalible [on Github!](http://github.com/germ/germ-footprints). 

Reviewer: `s/impaitent/impatient/`
`s/avalible/available/`

```suggestion
Or if you're impatient, build your own using the open source footprint available [on Github!](http://github.com/germ/germ-footprints). 
```

Also, I think this would read better if it were combined with the previous paragraph. (Basically, delete the two line breaks between this line and Line 31.
